For testing. Runs the test from a different environment than the Mozilla
default. Needs further investigation.

--- comm-esr31/mozilla/python/mozbuild/mozbuild/test/test_mozconfig.py.orig	2015-06-04 17:42:52.122873933 -0700
+++ comm-esr31/mozilla/python/mozbuild/mozbuild/test/test_mozconfig.py	2015-06-04 17:42:52.133296079 -0700
@@ -477,5 +477,5 @@
             self.assertEquals(e.exception.output, ['hello world'])
 
 
-if __name__ == '__main__':
-    main()
+#if __name__ == '__main__':
+#    main()
--- comm-esr31/mozilla/python/mozbuild/mozbuild/test/test_base.py.orig	2015-06-04 17:42:52.128263022 -0700
+++ comm-esr31/mozilla/python/mozbuild/mozbuild/test/test_base.py	2015-06-04 17:42:52.133425656 -0700
@@ -393,5 +393,5 @@
         self.assertEqual(p.srcdir_path(), "/src/foo/bar")
         self.assertEqual(p.objdir_path(), "/src/obj/foo/bar")
 
-if __name__ == '__main__':
-    main()
+#if __name__ == '__main__':
+#    main()
